content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/check_service_retired.rb

```diff
@@ -16,6 +16,13 @@ def main
 
               private
 
+              def log_task_info(task)
+                @handle.log('info', "Service Retire Task:<#{task.id}> <#{task.description}> is not retired, setting retry.")
+                task.miq_request_tasks.each do |t|
+                  @handle.log('info', "  Service Retire Task:<#{task.id}> waiting on <#{t.request_type}> Task:<#{t.id}> <#{t.description}>") if t.state != 'finished'
```

Comment:
`
@handle.log('info', "Service Retire Task <#{task.id}> is waiting on <#{t.request_type}> Task <#{t.id}> <#{t.description}>") ...
`

Reply:
Or we do want to keep the space in front of `Service Retire Task <#{task.id}> ...`?

Reply:
@lfu The spaces in front of are there so it's easier to see the 'waiting' tasks. The spaces really help if you have multiple vm's or a catalog bundle.  I'm not going to change this.